samples/android/HomeFragment.cs

```diff
@@ -71,28 +61,25 @@ public override void OnResume()
 		{
 			base.OnResume();
 
-			// Register a local broadcast manager to listen for ACTION_UPDATE_CHANNEL
-			LocalBroadcastManager localBroadcastManager = LocalBroadcastManager.GetInstance(Context);
-
-			// Use local broadcast manager to receive registration events to update the channel
-			IntentFilter channelUpdateFilter = new IntentFilter();
-			channelUpdateFilter.AddAction(UrbanAirshipReceiver.ACTION_CHANNEL_UPDATED);
-			localBroadcastManager.RegisterReceiver(channelIdUpdateReceiver, channelUpdateFilter);
+            UAirship.Shared().PushManager.AddRegistrationListener(this);
 
-			RefreshChannelId();
+            RefreshChannelId();
 		}
 
-		public override void OnPause()
-		{
-			base.OnPause();
-			LocalBroadcastManager localBroadcastManager = LocalBroadcastManager.GetInstance(Context);
-			localBroadcastManager.UnregisterReceiver(channelIdUpdateReceiver);
-		}
+        public override void OnPause()
+        {
+            base.OnPause();
 
-		void RefreshChannelId()
-		{
-			string channelIdString = UAirship.Shared().PushManager.ChannelId;
+            UAirship.Shared().PushManager.RemoveRegistrationListener(this);
+        }
 
+        void RefreshChannelId()
+        {
+            RefreshChannelId(UAirship.Shared().PushManager.ChannelId);
+        }
+
+		void RefreshChannelId(String channelIdString)
```

Comment:
indentation is also weird here. maybe consider autoformatting?

Reply:
tabs...